platform/src/main/resources/config/anchor-config-schema-v1.yaml

```diff
@@ -95,5 +98,4 @@ sep_server.context_path:
 sep_server.management_server_port:
 sep_server.port:
 stellar_network.horizon_url:
-stellar_network.network:
-stellar_network.network_passphrase:
```

Comment:
Is it something we want to remove in minor version?

Reply:
It does not break backward-compatibility. When specified, it is just ignored.